widget/embedded/src/containers/App/App.tsx

```diff
@@ -30,9 +30,6 @@ export function Main() {
     void useNotificationStore.persist.rehydrate();
     widgetContext.onConnectWallet(setLastConnectedWalletWithNetwork);
   }, []);
-  useEffect(() => {
-    changeLanguage(config?.language);
```

Comment:
It is impossible to alter the playground's language. Please review this line of code as it relates to it.